src/SFA.DAS.FAA.Application/Vacancies/Queries/GetSavedSearches/GetSavedSearchesQuery.cs

```diff
@@ -0,0 +1,7 @@
+﻿using MediatR;
+using System;
+
+namespace SFA.DAS.FAA.Application.Vacancies.Queries.GetSavedSearches
```

Comment:
I'm not convinced SavedSearches should be under vacancies namespace, since they're not directly related to vacancies themselves. Feels like SFA.DAS.FAA.Application.SavedSearches would be better?

Reply:
Corrected the namespace